notebooks/en/rag_with_hf_and_milvus.ipynb

```diff
@@ -0,0 +1,589 @@
+{
```

Comment:
<p>The RAG system combines a retrieval system with LLM. The system first retrieves relevant documents from a corpus using Milvus vector database, then uses an LLM hosted Hugging Face to generate answer based on the retrieved documents.</p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>